packages/opentelemetry-tracing/src/Tracer.ts

```diff
@@ -83,7 +81,7 @@ export class Tracer implements api.Tracer {
     }
     const spanKind = options.kind ?? api.SpanKind.INTERNAL;
     const links = options.links ?? [];
-    const attributes = { ...this._defaultAttributes, ...options.attributes };
+    const attributes = { ...options.attributes };
```

Comment:
No need for the destructure now.
```suggestion
    const attributes = options.attributes ?? {};
```